src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -41,13 +42,20 @@
     public static class VisionConstants {
         public double DISTANCE_CALCULATION_A_COEFFICIENT;
         public double DISTANCE_CALCULATION_B_COEFFICIENT;
-        // Offsets are measured from the robot's center of rotation to the limelight position.
+        // Offsets are measured from the robot's center of rotation to the limelight
+        // position.
         public double LIMELIGHT_OFFSET_X;
         public double LIMELIGHT_OFFSET_Y;
         public double LIMELIGHT_ANGLE_OFFSET;
         public double TARGET_NOT_FOUND_WAIT_TIME;
     }
 
+    public static class TrigonDriveConstents {
+        public double SENSITIVITY;
+        public double THRESHOLD;
+        public boolean X_LOCK;
```

Comment:
Remove all x lock occurrences. We use commands that require drivetrain instead of x locking